README.md

```diff
@@ -63,6 +65,8 @@ jobs:
     params:
       path: vault-concourse-resource/secret
       prefix: secret2
+      keys_to_copy: cert,pwd
```

Comment:
The param structure doesn't have to be flat, and so we can leverage actual YAML/JSON array to list the keys to copy.
```yaml
params:
  keys_to_copy:
  - cert
  - pwd
```
Also, we then don't have to rely on `,` as a sentinel character